website/blog/2021-02-04-64updates.md

```diff
@@ -0,0 +1,98 @@
+- [react-native-splash-screen](https://github.com/crazycodeboy/react-native-splash-screen) ([in PR](https://github.com/crazycodeboy/react-native-splash-screen/pull/514))
+
+… and more are coming! You can see which modules we’re tracking on the [Community Module Requests project board.](https://github.com/microsoft/react-native-windows/projects/23)
+
+In addition to the modules that we have recently worked on, you can view and search the complete list of modules that work with Windows by visiting [reactnative.directory](https://reactnative.directory/). The directory catalogues modules that work on the major platforms including iOS, Android, Windows, and macOS and provides metrics around usage, popularity, and more.
+
+You can also check out the [react-native-gallery repo](https://github.com/microsoft/react-native-gallery) to see some of the modules “in action”. We’re using the Gallery as a testbed to verify and showcase modules that work with Windows – we’re adding new samples over time, so check back often!
+
+## 🚧 A module I need doesn't work with Windows yet!
+
+Fret not! We’re continually reassessing and reviewing the set of modules that are important to React Native for Windows devs, so if you need a module that doesn’t yet have a Windows implementation, please [open a new issue on the React Native for Windows GitHub repo.](https://github.com/microsoft/react-native-windows/issues/new/choose)
+
+If you have experience writing Windows native code, you can also use the [new Community module template workflow](https://microsoft.github.io/react-native-windows/docs/next/native-modules-setup) to create a module template for Windows. This is a great way to start using native Windows APIs quickly in custom modules.
+
+## 🤝 Janea Systems
+![janea-logo](assets/janeasystemslogo.png)
+
+Finally, we wanted to highlight our partnership with [Janea Systems](http://www.janeasystems.com/), who has helped provide implementation, consultation, and support for many of the modules listed above. In particular, we wanted to shout out to [@jaimecbernardo](https://github.com/jaimecbernardo), [@joaocgreis](https://github.com/joaocgreis), and [@bzoz](https://github.com/bzoz) from Jeana Systems for their help over the last few months.
+
+# 🍎 React Native for macOS
+Community Work
+
+- React-Native-Community macOS ProgressView support
+- React-Native-Community macOS Clipboard support
+- React-Native-Community macOS SVG Icon support
+- We can’t check this in because the repo owner has to merge it and he is just one person who is “taking a break” from OSS, but it’s up and running in our fork.
+We’re setting up the npm publishing CI this week for macOS
+
+- macOS keyboarding events support
+- macOS Tooltip support for Button, Text, Image, Slider, and TextInput
+This is out for review now but will get checked in tomorrow

```

Comment:
remove this line?

Reply:
It should be removed, it's a stub at the moment. I need more macOS info. Do you know who to talk to about it?

Reply:
Chris Hogan is likely to have the most context after Eloy, who is no longer working on RN.

I don't think there are many marketable changes for macOS beyond reaching 0.63. Mostly just some Office-specific internal fixes. Don't think we will be able to get a lot of content here.

Reply:
Then perhaps I should just omit it for this blog.